src/status_im/feature_flags.cljs

```diff
@@ -0,0 +1,32 @@
+(ns status-im.feature-flags
+  (:require
+    [react-native.config :as config]
+    [reagent.core :as reagent]))
+
+(defn- enabled? [v] (= "1" v))
+
+(defn check-env [k] (enabled? (config/get-config k)))
+
+(def ^:private feature-flags-config
+  (reagent/atom
+   {:wallet
+    {:edit-default-keypair (check-env :DEV_FF_EDIT_DEFAULT_KEYPAIR)
```

Comment:
should allow for defaults to be set by an env var so devs can have a better dev experience - didn't quite get this working for new vars, perhaps dependent on the app being built for it to load?
@siddarthkay do you know?
better again would be that we can use env.dev or something hidden by gitignore 👍

Reply:
looks good to me overall,
> should allow for defaults to be set by an env var so devs can have a better dev experience - didn't quite get this working for new vars, perhaps dependent on the app being built for it to load?

I think that's desireable,
if you need to pull from env, you will have to rebuild the code with the new env variable set, since they are build time flags, I think. 

We have already a bunch of feature flags in `config` maybe worth moving those as well at some point?

the only thing that is missing that I can see, is that some feature flags are not UI only (anything that is also toggling status-go code will want for example to be making potentially an RPC and store it in settings, so you have it on when you login/logout/restart etc), maybe we should accommodate for that use case (some hooks or something)? (@alwx is working on adding a feature flag for some status-go code).

Also worth considering if you want to store in async storage, just to make it a bit more permanent, it might be a bit unexpected for a developer that `make run..` clears your feature flags (unless you set them in the env file), but either is fine I think, this is simpler.